### PR TITLE
feat: Refactor vue components to use i18n

### PR DESCRIPTION
**Pull request by lipskij**
Refactor vue components to use internalization

### CODE REVIEW

**Review by nerdCopter — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> PASS
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Pull request merged**
